schema/application/resolvers/Mutation/discardChecklist.ts

```diff
@@ -0,0 +1,54 @@
+import { sql } from "@/datasources/postgres";
+import type { MutationResolvers } from "@/schema";
+import { decodeGlobalId } from "@/schema/system";
+import { GraphQLError } from "graphql";
+import { copyFromWorkTemplate } from "./copyFrom";
+
+export const discardChecklist: NonNullable<
+  MutationResolvers["discardChecklist"]
+> = async (_, { entity }) => {
+  const { type, id } = decodeGlobalId(entity);
+
+  if (type !== "workinstance") {
+    throw new GraphQLError("Entity cannot be discarded.", {
+      extensions: {
+        code: "E_INVALID_STATE_CHANGE",
+      },
+    });
+  }
+
+  const result = await sql.begin(async tx => {
+    const data = await tx`
+        WITH inputs AS (
+            SELECT systagid AS status
+            FROM public.systag
+            WHERE
+                systagparentid = 705
+                AND
+                systagtype = 'Cancelled'
+        ),
+        updated_instance AS (
+            UPDATE public.workinstance
+            SET
+                workinstancestatusid = inputs.status,
+                workinstancemodifieddate = now()
```

Comment:
do we need `workinstancemodifiedby`? passing in the worker id to get a worker instance id should be easy